### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by IlyaGrynyshyn**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cp, input_file, output_file = command.split(" ")
```
Comment (pavlejviki) [inline app/main.py:2]: cp is not a descriptive variable name

Comment (pavlejviki) [inline app/main.py:2]: make sure that the command consists of three parts

**Review by pavlejviki — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 2]
```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split(" ")
+    if len(command) == 3:
+        function, input_file, output_file = command
+        if function != function or input_file == output_file:
```
Comment (viktoria-rybenchuk) [inline app/main.py:5]: what do you check by this `if function != function`. You should check if command is `cp`

**Review by viktoria-rybenchuk — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 3]
```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split(" ")
+    if len(command) == 3:
+        function, input_file, output_file = command
```
Comment (viktoria-rybenchuk) [inline app/main.py:4]: this is not a `function` it is command. Write correct variable names)

**Review by viktoria-rybenchuk — CHANGES_REQUESTED** [review]

**Review by pavlejviki — APPROVED** [review]
Good job!

**Pull request closed without merging**